lib/dl_api_commons/dl_api_commons_tests/unit/test_reporting.py

```diff
@@ -210,7 +210,6 @@ def test_db_query_report_generation(case_name, records_seq, expected_query_data,
         "event_code",
         "dataset_id",
         "user_id",
-        "billing_folder_id",
```

Comment:
Let's add a tenant (for example, `TenantCommon`) to the test data [here](http://github.com/datalens-tech/datalens-backend/blob/a34d3fc6185fb2db784f512d67832abe42144f14/lib/dl_api_commons/dl_api_commons_tests/unit/test_reporting.py#L86-L86), and modify the test accordingly.